### ISSUE TITLE
Suggestion: Ability to rename or name

### ISSUE THREAD

**Issue by chef-danny-d** [inns/mak-mini#5]
It would be nice to have the ability to rename or sync gists under different names since it can get hard to manage notes with duplicate names.

**Comment by quietshu** [comment]
Yes this could be really helpful. Need to find a proper way to manage those files. Maybe the #list needs to be redesigned with some more buttons... 
Anyway I will think about it and try to come out with an idea on this.

Thanks for the suggestion!
